falco/Chart.yaml

```
@@ -17,3 +17,9 @@ sources:
 maintainers:
   - name: The Falco Authors
     email: cncf-falco-dev@lists.cncf.io
+dependencies:
+  - alias: falcosidekick
+    name: falcosidekick
+    version: 0.2.2
```

```suggestion
    version: "0.2.*"
```
Use a wildcard for patch version, as discussed privatelyly